LoopKit/PumpManager.swift

```diff
@@ -50,11 +50,32 @@ public protocol PumpManagerDelegate: PumpManagerStatusObserver {
 
 
 public protocol PumpManager: DeviceManager {
-    // Rounds units to the nearest delivery increment
-    func roundToDeliveryIncrement(units: Double) -> Double
+    /// Rounds a temporary basal rate in U/hr to a rate supported by this pump.
+    ///
+    /// - Parameters:
+    ///   - unitsPerHour: A desired rate of delivery in Units/hr
+    /// - Returns: a supported rate of delivery in Units/hr. The rate returned should not be larger than the passed in rate.
+    func roundToSupportedTemporaryBasalRate(unitsPerHour: Double) -> Double
```

Comment:
[nit] Remove the "Temporary" aspect here. Just `roundToSupportedBasalRate` for consistency.